packages/secure_storage/amplify_secure_storage_dart/lib/src/ffi/cupertino/core_foundation.bindings.g.dart

```diff
@@ -67,19 +69,6 @@ class CoreFoundation {
           ffi.Pointer<CFDictionaryKeyCallBacks>,
           ffi.Pointer<CFDictionaryValueCallBacks>)>();
 
-  void CFRelease1(
```

Comment:
These were duplicates generated by ffigen. Good to see they are no longer being generated 🔥